pbj-integration-tests/src/main/java/module-info.java

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Moved `requires` statements from `build.gradle.kts` to real `module-info` file to create a test setup where one module – `com.hedera.pbj.integration.tests.custom` – can require another one  – `com.hedera.pbj.integration.tests.custom` – and depend on protobufs from that one.